android/src/ti/barcode/BarcodeModule.java

```diff
@@ -296,10 +296,11 @@ public void capture(@Kroll.argument(optional = true) HashMap args) {
 
 			intent.putExtra(Intents.Scan.SHOW_RECTANGLE, argsDict.optBoolean("showRectangle", true));
 			intent.putExtra(Intents.Scan.KEEP_OPEN, argsDict.optBoolean("keepOpen", false));
-
+			frameWidth = argsDict.optInt("frameWidth", 0);
+			frameHeight = argsDict.optInt("frameHeight",0);
```

Comment:
If I got it right this will define the frame in where the barcode is supposed to be placed? If that is correct it will be good to include this in the docs as well.